### PR TITLE
Add schemas configuration and description in readme

### PR DESCRIPTION
**Pull request by darkBasic666**

### CODE REVIEW

**Comment by mishuagopian** [comment]
@darkBasic666 would you fix conflicts, please?

File: config/development.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ exports.config = {
       port: process.env.NODE_API_DB_PORT,
       database: process.env.NODE_API_DB_NAME,
       username: process.env.NODE_API_DB_USERNAME,
-      password: process.env.NODE_API_DB_PASSWORD
+      password: process.env.NODE_API_DB_PASSWORD,
```
Comment (gonziesc) [inline config/development.js]: We abstracted this to `index` in config, you will just need to add this there

File: app/models/book.js [hunk 2]
```diff
@@ -1,4 +1,15 @@
-const Sequelize = require('sequelize');
+const Sequelize = require('sequelize'),
+  config = require('./../../config/');
+
+const options = {
```
Comment (gonziesc) [inline app/models/book.js]: We could abstract this to some other file so we don't repeat logic here

File: migrations/migrations/20170526115728-user-create.js [hunk 3]
```diff
@@ -1,41 +1,47 @@
...
-      password: {
-        type: Sequelize.STRING,
-        allowNull: false
-      },
-      created_at: Sequelize.DATE,
-      updated_at: Sequelize.DATE,
-      deleted_at: Sequelize.DATE
-    });
+      {
+        schema: process.env.NODE_API_DB_SCHEMA
```
Comment (gonziesc) [inline migrations/migrations/20170526115728-user-create.js]: This is added automatically with any migration script config or we have to add it manually?

**Comment by gonziesc** [comment]
@darkBasic666 Left some comments here!

File: migrations/migrations/20170526115728-user-create.js (hunk 3 above)
Comment (darkBasic666) [reply migrations/migrations/20170526115728-user-create.js]: manually

**Pull request closed without merging**